database/migrations/2025_05_11_030824_add_removal_reason_and_deleted_by_to_waiting_list_account.php

```diff
@@ -0,0 +1,30 @@
+<?php
+
+use Illuminate\Database\Migrations\Migration;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Support\Facades\Schema;
+
+return new class extends Migration
+{
+    /**
+     * Run the migrations.
+     */
+    public function up(): void
+    {
+        Schema::table('training_waiting_list_account', function (Blueprint $table) {
+            $table->unsignedInteger('removed_by')->nullable();
+            $table->text('removal_reason')->nullable();
```

Comment:
We should be specific about which columns these are going after by using ->after()